src/components/donation-flow/DonationFlowPage.tsx

```diff
@@ -0,0 +1,137 @@
+import Link from 'next/link'
+import Image from 'next/image'
+import { styled } from '@mui/material/styles'
+import { Box, Grid, Typography, useMediaQuery } from '@mui/material'
+
+import theme from 'common/theme'
+import { routes } from 'common/routes'
+import {
+  backgroundCampaignPictureUrl,
+  beneficiaryCampaignPictureUrl,
+} from 'common/util/campaignImageUrls'
+import Layout from 'components/layout/Layout'
+import { useViewCampaign } from 'common/hooks/campaigns'
+import CenteredSpinner from 'components/common/CenteredSpinner'
+// import RadioAccordionGroup, { testRadioOptions } from 'components/donation-flow/RadioAccordionGroup'
+// import PaymentDetailsStripeForm from 'components/donations/stripe/PaymentDetailsStripeForm'
+
```

Comment:
This page is a copy of the `OneTimeDonationPage.tsx`, but with the DonationStepper removed. Since the layout of the page banner is the same as the old donation flow.